### DISCUSSION TITLE
Feature: select shows/episodes from UI

### DISCUSSION THREAD

**Discussion by robertolopezlopez** [advplyr/audiobookshelf-app#1249] [Q&A] [answered]
Hi, 

I've started using server (docker Linux server images) and android app short time ago. I want to thank you for such great free software. 

The only problem which I've found is in the app. I would expect to be able to select shows and episodes the same way as I can do in the server web UI. Is this a known problem or (hopefully) an upcoming feature?

![Screenshot_20240626-083935.png](https://github.com/advplyr/audiobookshelf-app/assets/1519467/f04d59b7-e6d0-467f-aa18-a80d266f1535)

Thank you! :)

**Comment by nichwall** [comment, marked as answer]
It is planned.
https://github.com/advplyr/audiobookshelf-app/issues/188

**Reply by robertolopezlopez** [reply]
ah, awesome @nichwall  - thanks!
